pkg/fleet/installer/default_packages.go

```diff
@@ -46,15 +54,22 @@ func defaultPackages(env *env.Env, defaultPackages []defaultPackage) []string {
 		if isOverridden {
 			shouldInstall = installOverride
 		}
+		if !shouldInstall {
+			continue
+		}
+
+		version := "latest"
+
+		// Respect pinned version of APM packages if we don't define any overwrite
+		if apmLibVersion, ok := env.ApmLibraries[languageToPackage[p.name]]; ok {
+			version = apmLibVersion.AsVersionTag()
+		}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏With the default in `AsVersionTag` I think we can remove the condition